src/test/java/seedu/duke/command/DeleteCommandTest.java

```diff
@@ -47,13 +49,13 @@ public void execute_emptyList_throwsItemNotExitException() {
     }
 
     @Test
-    public void execute_noMatchedItemInList_throwsItemNotExitException() throws Exception {
+    public void execute_noMatchedItemInList_throwsItemNotExitException() throws DuplicateItemException {
         testList.addItem(testItem1);
         assertThrows(ItemNotExistException.class, () -> testCommand2.execute(testList));
```

Comment:
i assume it is a typo line 52 throwsItemNotExitException and line 54 ItemNotExistException